packages/liveblocks-react/src/factory.tsx

```diff
@@ -296,9 +296,9 @@ export function createRoomContext<
       setRoom(room);
 
       return () => {
-        const commentsRoom = commentsRooms.get(roomId);
+        const commentsRoom = commentsRooms.get(room);
```

Comment:
The `roomId` is a string, whereas the `room` is a Room instance. How did TypeScript accept this before?

Reply:
The `commentsRoom` map used to be `Map<string, CommentsRoom>` before this PR. https://github.com/liveblocks/liveblocks/pull/1150/files#r1334019235